packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -298,6 +298,12 @@ export class Cluster extends Resource implements ICluster {
    */
   public readonly clusterCertificateAuthorityData: string;
 
+  /**
+   * The OpenID Connect Issuer Url for your cluster.
+   * @attribute
+   */
+  public readonly clusterOpenIdConnectIssuerUrl?: string;
```

Comment:
Why is this optional (attributes are always defined)?

Reply:
The CfnCluster resource doesn't expose this attribute. I can only resolved using the API. This is why kubectlEnabled must be true. Only then it can get the OpenIdConnectProvider.
An alternative would be to use a custom resource for just retrieving the Issuer URL.